launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/BuildScaffolder.kt

```diff
@@ -44,22 +39,25 @@ class BuildScaffolder(private val launcher: Launcher) {
 
     private val savePropsFlag get() = launcher.args.contains(Launcher.ARG_SAVE_PROPS)
 
-    private val saveProps get() = launcher.args.filter { it.startsWith(Launcher.ARG_SAVE_PREFIX) }
-        .map { it.removePrefix(Launcher.ARG_SAVE_PREFIX) }
-        .map { it.substringBefore("=") to it.substringAfter("=") }
-        .toMap()
+    private val saveProps
+        get() = launcher.args.filter { it.startsWith(Launcher.ARG_SAVE_PREFIX) }
+            .map { it.removePrefix(Launcher.ARG_SAVE_PREFIX) }
+            .map { it.substringBefore("=") to it.substringAfter("=") }
+            .toMap()
 
-    private val defaultProps get() = mapOf(
-        "org.gradle.logging.level" to "info",
-        "org.gradle.daemon" to "true",
-        "org.gradle.parallel" to "true",
-        "org.gradle.caching" to "true",
-        "org.gradle.jvmargs" to "-Xmx2048m -XX:MaxPermSize=512m -Dfile.encoding=UTF-8"
-    )
+    private val defaultProps
+        get() = mapOf(
+            "org.gradle.logging.level" to "info",
+            "org.gradle.daemon" to "true",
+            "org.gradle.parallel" to "true",
+            "org.gradle.caching" to "true",
+            "org.gradle.jvmargs" to "-Xmx2048m -XX:MaxPermSize=512m -Dfile.encoding=UTF-8"
```

Comment:
please delete that `-XX:MaxPermSize=512m` as it is incompatible with recent Java